tests/e2e/utils/requestHandlers/headers/CheMultiuserAuthorizationHeaderHandler.ts

```diff
@@ -8,20 +8,14 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 import { AxiosRequestConfig } from 'axios';
-
-import { TYPES } from '../../../inversify.types';
 import { IAuthorizationHeaderHandler } from './IAuthorizationHeaderHandler';
-import { injectable, inject } from 'inversify';
-import { ITokenHandler } from '../tokens/ITokenHandler';
+import { injectable } from 'inversify';
 
 @injectable()
 export class CheMultiuserAuthorizationHeaderHandler implements IAuthorizationHeaderHandler {
 
-    constructor(@inject(TYPES.ITokenHandler) private readonly tokenHandler: ITokenHandler) {
-    }
-
     async get(): Promise<AxiosRequestConfig> {
-        const token = await this.tokenHandler.get();
-        return { headers: { 'Authorization': `Bearer ${token}` } };
+        //  to-do : Fetch the cookies from user api and pass it here
```

Comment:
@makambalaji  Do the tests still work without this logic in place? Are we sure we can leave it as WIP? Is there a follow-up issue?

Reply:
This token is used mainly for the api related functions [Eg: PreferenceHandler functions], which are not using in spec files any more. so it won't be any issue
Follow up issue : https://github.com/eclipse/che/issues/21203

Reply:
hmm, refer to the previous comment